packages/codegen-ui-react/lib/__tests__/__snapshots__/studio-ui-codegen-react-forms.test.ts.snap

```diff
@@ -379,6 +379,11 @@ export default function CreateCompositeDogForm(props) {
           modelFields = onSubmit(modelFields);
         }
         try {
+          Object.entries(modelFields).forEach(([key, value]) => {
+            if (value === \\"\\") {
```

Comment:
Should we trim string in case the user presses the space bar? `if value.trim() === ""`

Reply:
Makes sense. Will do